lib/ex_doc/formatter/html/templates/head_template.eex

```diff
@@ -2,18 +2,22 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
```

Comment:
`meta` is a void element, according to the HTML5 spec the `/` character has no effect.